pkg/core/resources/model/utils.go

```diff
@@ -96,3 +96,11 @@ func IsEmpty(spec ResourceSpec) bool {
 func PtrTo[T any](t T) *T {
 	return &t
 }
+
+func SafeDeref[T any](ptr *T) T {
```

Comment:
Can we try to have this stuff in separate PRs? This has nothing to do with `MeshAccessLog`

Reply:
Probably it can be a separate PR, but it should go before this because this func is actively used

Reply:
https://github.com/kumahq/kuma/pull/5586